scripts/rewrite/resources/post-to-canny.py

```
@@ -0,0 +1,245 @@
+        "apiKey":   CANNY_API_KEY,
+        "authorID": CANNY_ADMIN_ID,
+        "boardID":  CANNY_BOARD_ID,
+        "title":    full_title,
+        "details":  body,
+    }).encode()
+    req = urllib.request.Request(
+        "https://canny.io/api/v1/posts/create",
+        data=payload, method="POST",
+    )
+    try:
+        with urllib.request.urlopen(req, timeout=20) as resp:
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)